includes/controls/icons.php

```diff
@@ -155,6 +159,9 @@ protected function get_default_settings() {
 			'recommended' => false,
 			'is_svg_enabled' => Svg_Handler::is_enabled(),
 			'skin' => 'media',
+			'inline_options' => [
+				'exclude' => [],
+			],
```

Comment:
Do it like the following example:
```
'inline_options' => [
    'icon',
    'svg',
],
```

Reply:
It doesn't work. because the override settings don't fully _overwrite_ the default settings - it happens with `array_merge` (`\includes\controls\base.php` in the constructor).
So if you only include one of the two options (`icon` or `svg`) in the widget's control instance - the `array_merge` will add the other one. 

It has to be a nested array.